sdk/lib/atl/atldef.h

```diff
@@ -7,6 +7,15 @@
 
 #pragma once
 
+#ifdef NDEBUG
+    #undef DBG
```

Comment:
DBG is a systemwide macro that tells whether code is compiled in debug mode or not. Disabling it that way (especially wrt. NDEBUG) is not safe.